src/map/script.c

```diff
@@ -9013,6 +9013,32 @@ BUILDIN(getgmlevel)
 	return true;
 }
 
+/// set the group ID of the player.
+/// setgroupid( <new group id>,{<account ID> or <Player Name>} )
+/// return 1 on success, 0 if failed.
+BUILDIN(setgroupid) {
+	TBL_PC* sd;
```

Comment:
`TBL_PC *sd == NULL;`
yes, the standard for pull request is strict ...
I also prefer to write plugins rather than pull request ...